spec/models/application_reference_spec.rb

```diff
@@ -90,6 +90,43 @@
     end
   end
 
+  describe '#failed?' do
+    let(:failed_states) { %w[feedback_refused cancelled cancelled_at_end_of_cycle] }
+
+    context 'with failed states' do
+      it 'returns true' do
+        failed_states.each do |state|
+          reference.update!(feedback_status: state)
+          expect(reference.failed?).to be true
```

Comment:
fwiw i think rspec magic will give you the beauty of `expect(reference).to be_failed?`